### DISCUSSION TITLE
Aggregate responses from multiple LLMs

### DISCUSSION THREAD

**Discussion by odefta** [open-webui/open-webui#5108] [General]
**Current Situation**
In a chat session, we can select multiple LLMs to respond to a prompt. Each LLM provides its own response, so we end up with multiple separate answers.

**Proposal**
It would be great to have the option to use another LLM model to automatically aggregate these responses into a single, unified answer. This would be particularly useful for tackling complex logical problems, even though it may result in higher token usage.

**Background**
I’ve tested the "Mixture of Agents" function, but it seems to only work with local models, limiting its flexibility.

**Comment by cheahjs** [comment]
Already a feature introduced in v0.3.14.

<img width="136" alt="image" src="https://github.com/user-attachments/assets/7d5b0d84-9a63-4c16-b93b-ff27eadddba3">

**Comment by odefta** [comment]
Thanks, awesome! I had an earlier version 0.3.12. Do you know which model will be used to display the (aggregated) final answer?

**Comment by odefta** [comment]
Can we choose which model is used for aggregation and / or which prompt is used for instructions?
